src/clients/inst_autosetup.rb

```diff
@@ -183,10 +183,13 @@ def main
 
       if Builtins.haskey(Profile.current, "add-on")
 	Progress.Title(_("Handling Add-On Products..."))
-        Call.Function(
-          "add-on_auto",
-          ["Import", Ops.get_map(Profile.current, "add-on", {})]
-        )
+        unless Call.Function(
+            "add-on_auto",
+            ["Import", Ops.get_map(Profile.current, "add-on", {})]
+          )
+          log.warn("User has aborted the installation.")
```

Comment:
NP: I'd just add an empty line here, it's not that clear where the `unless` condition ends and where starts the body.